### ISSUE TITLE
Feature request: theme preview UI

### ISSUE THREAD

**Issue by mikemaccana** [microsoft/Terminal#706]
You're probably already working on this, but it would be useful to have an issue to coordinate discussion.

@felixse's Fluent Terminal has a pretty lovely theme UI that gets it 90% right:

![image](https://user-images.githubusercontent.com/172594/57571662-d5616100-7408-11e9-9763-7c72ccca110d.png)

Some good things about it:

 - Previews of everything, rather than just the theme name
 - Users can click on the colors to edit 
 - It can import .itermcolors directly, rather than asking users to run `colortool`

One thing I'd add: the ability to just type in the name of [the many well known color schemes](https://iterm2colorschemes.com/) preview it, and have it installed. This saves having to download a color scheme, convert it, and put it in a directory and import it.

**Comment by 5HT** [comment]
Somebody could prefer single Dialog window with settings per entire terminal rather that complex hierarchical menus. Even highly customized xterm lacks fancy settings.

**Comment by mikemaccana** [comment]
@SHT Agreed, but there's only one layer of heirarchy here (with 6 options, 7 if you include about). This Issue is about theming through, and doesn't suggest anything about how Terminal Settings should be laid out.

**Comment by SHT** [comment]
@5HT (just in case you don't see this because of the similar nicknames)

**Comment by DHowett-MSFT** [comment]
We've got some pretty cool (we think) designs in mind for the settings UI that should assuage a lot of these concerns.

**Comment by mdtauk** [comment]
Whilst we are throwing around ideas  for settings and theme previews.

![image](https://user-images.githubusercontent.com/7389110/57575866-e7fc8a00-744a-11e9-82e1-dea12304276e.png)

The simpler the better, with a visual preview, and simple colour entry.  the "Add new profile" button could either open a blank profile screen for choosing terminal, or offer the choice to import a profile as well as create a new one.

**Comment by DHowett-MSFT** [comment]
@mdtauk,
Thanks for the input! We’ll make sure it gets back to our designers.
